[2015-05-13 08:20:50] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\routes.php:24
Stack trace:
#0 {main}  
[2015-05-13 08:20:59] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\KhoaController does not exist' in D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php:1029
Stack trace:
#0 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1029): ReflectionClass->__construct('App\\Http\\Contro...')
#1 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(980): Illuminate\Container\Container->build('App\\Http\\Contro...', Array)
#2 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1493): Illuminate\Container\Container->make('App\\Http\\Contro...', Array)
#3 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8327): Illuminate\Foundation\Application->make('App\\Http\\Contro...')
#4 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8316): Illuminate\Routing\ControllerDispatcher->makeController('App\\Http\\Contro...')
#5 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(7317): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#6 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(7288): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#7 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6954): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6955): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6944): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#14 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#17 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(2438): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#20 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#25 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#34 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2015-05-13 08:21:37] local.ERROR: exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'qlsinhvien.users' doesn't exist' in D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Connection.php:288
Stack trace:
#0 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Connection.php(288): PDO->prepare('select * from `...')
#1 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Connection.php(612): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'select * from `...', Array)
#2 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Connection.php(576): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#3 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Connection.php(293): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#4 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1377): Illuminate\Database\Connection->select('select * from `...', Array, true)
#5 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1367): Illuminate\Database\Query\Builder->runSelect()
#6 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1354): Illuminate\Database\Query\Builder->getFresh(Array)
#7 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(380): Illuminate\Database\Query\Builder->get(Array)
#8 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(162): Illuminate\Database\Eloquent\Builder->getModels(Array)
#9 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(136): Illuminate\Database\Eloquent\Builder->get(Array)
#10 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(724): Illuminate\Database\Eloquent\Builder->first()
#11 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(514): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#12 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1985): Illuminate\Auth\Guard->attempt(Array, false)
#13 [internal function]: App\Http\Controllers\Auth\AuthController->postLogin(Object(Illuminate\Http\Request))
#14 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8287): call_user_func_array(Array, Array)
#15 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8354): Illuminate\Routing\Controller->callAction('postLogin', Array)
#16 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8333): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\Auth\AuthController), Object(Illuminate\Routing\Route), 'postLogin')
#17 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#18 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\Middleware\RedirectIfAuthenticated.php(41): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): App\Http\Middleware\RedirectIfAuthenticated->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#23 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8334): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#24 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8320): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\Auth\AuthController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'postLogin')
#25 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(7317): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'postLogin')
#26 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(7288): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#27 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6954): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#29 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6955): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#33 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6944): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#34 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#35 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#36 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#37 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#38 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(2438): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#40 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#49 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#52 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#53 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#54 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#55 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#56 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'qlsinhvien.users' doesn't exist (SQL: select * from `users` where `email` = nguyenhuuhientv@gmail.com limit 1)' in D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Connection.php:620
Stack trace:
#0 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Connection.php(576): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#1 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Connection.php(293): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#2 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1377): Illuminate\Database\Connection->select('select * from `...', Array, true)
#3 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1367): Illuminate\Database\Query\Builder->runSelect()
#4 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1354): Illuminate\Database\Query\Builder->getFresh(Array)
#5 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(380): Illuminate\Database\Query\Builder->get(Array)
#6 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(162): Illuminate\Database\Eloquent\Builder->getModels(Array)
#7 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(136): Illuminate\Database\Eloquent\Builder->get(Array)
#8 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(724): Illuminate\Database\Eloquent\Builder->first()
#9 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(514): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#10 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1985): Illuminate\Auth\Guard->attempt(Array, false)
#11 [internal function]: App\Http\Controllers\Auth\AuthController->postLogin(Object(Illuminate\Http\Request))
#12 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8287): call_user_func_array(Array, Array)
#13 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8354): Illuminate\Routing\Controller->callAction('postLogin', Array)
#14 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8333): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\Auth\AuthController), Object(Illuminate\Routing\Route), 'postLogin')
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\Middleware\RedirectIfAuthenticated.php(41): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): App\Http\Middleware\RedirectIfAuthenticated->handle(Object(Illuminate\Http\Request), Object(Closure))
#19 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#21 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8334): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#22 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8320): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\Auth\AuthController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'postLogin')
#23 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(7317): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'postLogin')
#24 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(7288): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#25 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6954): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#27 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#30 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6955): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#31 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6944): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#32 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#33 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#35 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#36 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(2438): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#38 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#49 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#51 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#52 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#53 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#54 {main}  
[2015-05-13 08:28:33] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\Auth\AuthController does not exist' in D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php:8420
Stack trace:
#0 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(8420): ReflectionClass->__construct('App\\Http\\Contro...')
#1 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6787): Illuminate\Routing\ControllerInspector->getRoutable('App\\Http\\Contro...', 'auth')
#2 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6778): Illuminate\Routing\Router->controller('auth', 'Auth\\AuthContro...')
#3 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(5301): Illuminate\Routing\Router->controllers(Array)
#4 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\routes.php(21): Illuminate\Support\Facades\Facade::__callStatic('controllers', Array)
#5 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\routes.php(21): Illuminate\Support\Facades\Route::controllers(Array)
#6 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(17158): require('D:\\xampp\\htdocs...')
#7 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#8 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(6819): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#9 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(17159): Illuminate\Routing\Router->group(Array, Object(Closure))
#10 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#11 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(925): call_user_func_array(Array, Array)
#12 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(4858): Illuminate\Container\Container->call(Array)
#13 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(4840): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#14 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(17153): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#15 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#16 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(925): call_user_func_array(Array, Array)
#17 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1518): Illuminate\Container\Container->call(Array)
#18 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1510): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#19 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 18)
#20 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1511): array_walk(Array, Object(Closure))
#21 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1785): Illuminate\Foundation\Application->boot()
#22 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1311): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#23 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1928): Illuminate\Foundation\Application->bootstrapWith(Array)
#24 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1890): Illuminate\Foundation\Http\Kernel->bootstrap()
#25 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#26 D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#27 {main}  
[2015-05-13 08:29:42] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\routes.php:24
Stack trace:
#0 {main}  
[2015-05-13 08:35:48] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\Khoa' not found' in D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\Controllers\KhoaController.php:33
Stack trace:
#0 {main}  
[2015-05-13 08:35:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\App\Khoa' not found' in D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\Controllers\KhoaController.php:33
Stack trace:
#0 {main}  
[2015-05-13 08:40:40] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected end of file' in D:\xampp\htdocs\ql_sinhvien\qlsinhvien\trunk\app\Http\routes.php:26
Stack trace:
#0 {main}  
